src/darwin/Framework/CHIP/CHIPCluster.h

```diff
@@ -57,6 +57,21 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (strong, nonatomic, nullable) NSNumber * timedWriteTimeoutMs;
 
+/**
+ * Sets the data version for the Write Request for the interaction.
+ *
+ * This tag can be omitted if the value of EnableTagCompression in the Path field is true.
+ * In this case, the value for the omitted tag SHALL be set to the value for that
+ * tag (if present) in the last AttributeDataIB that had tag compression disabled
+ * and was seen in a message that is part of the same interaction model Action
+ * as the current message.
+ *
+ * If this tag was not present and tag compression was disabled,
+ * it SHALL be interpreted as though a data version was not specified in that,
+ * or subsequent AttributeDataIBs.
```

Comment:
This comment makes no sense as API documentation here.  What is "EnableTagCompression"?  That's a low-level concept this API does not know about.

How about:
```suggestion
 * If not nil, the write will only succeed if the current data version of the cluster matches the provided data version.
```

Reply:
Updated.